dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java

```diff
@@ -31,6 +32,7 @@ public static void main(String[] args) {
     }
 
     @Component
+    @Profile("create")
```

Comment:
InitRunner` also needs to add profile annotations, otherwise it will also be executed.

Reply:
I will fix it.

Reply:
Do we need to keep InitDolphinScheduler and UpgradeDolphinScheduler? Maybe CreateDolphinScheduler is enough.
What do you think?

Reply:
> Do we need to keep InitDolphinScheduler and UpgradeDolphinScheduler? Maybe CreateDolphinScheduler is enough. What do you think?

I think so.

Reply:
> UpgradeDolphinScheduler

I think it's ok, but please change the name of CreateDolphinScheduler to UpgradeDolphinScheduler.